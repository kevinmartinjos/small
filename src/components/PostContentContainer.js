import React from 'react';
import ReactHtmlParser from 'react-html-parser';
import '../styles/PostContentContainer.css';

var PropTypes = React.PropTypes;

var PostContentContainer = React.createClass({
	propTypes: {
		content: PropTypes.string.isRequired,
		onMouseUp: PropTypes.func
	},
	render() {
		return(
			<div className="PostContentContainer">
				<section>{ReactHtmlParser(this.props.content)}</section>
			</div>
		);
	}
});

export default PostContentContainer;
