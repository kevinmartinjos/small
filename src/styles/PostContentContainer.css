.PostContentContainer section{
	font-size: 18px;
	line-height: 1.58;
	letter-spacing: -.004em;
	font-family: 'medium-content-serif-font,Georgia,Cambria,"Times New Roman",Times,serif'
}

.PostContentContainer .blink{
	animation: blinker 1s linear 2;
}

@keyframes blinker {  
  50% { opacity: 0; }
}