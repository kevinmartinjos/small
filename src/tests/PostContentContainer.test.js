import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import PostContentContainer from '../components/PostContentContainer';
import ReactTestUtils from 'react-addons-test-utils';

test('PostContentContainer should be rendered', () => {
	const component = renderer.create(
		<PostContentContainer content="I am a post" />
	);
	let tree = component.toJSON();
  	expect(tree).toMatchSnapshot();
});
