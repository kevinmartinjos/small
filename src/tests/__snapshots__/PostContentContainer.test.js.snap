exports[`test PostContentContainer should be rendered 1`] = `
<div
  className="PostContentContainer">
  <section>
    I am a post
  </section>
</div>
`;
